stories/map.stories.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withInfo } from "@storybook/addon-info";
 
-import Map, { Map as MapWrapped } from '../app/js/components/Map';
```

Comment:
The same way we have stories in seller directory, should you also move buyer into his own directory